src/cli.cr

```diff
@@ -2,7 +2,80 @@ require "dppm/cli"
 require "./dppm_rest_api"
 
 module DppmRestApi::CLI
-  def self.run_server(config, host, port, data_dir, **args)
+  extend self
+
+  # A helper method for the select_users method -- splits a comma-separated
+  # list of numbers in a string into an `Array` of `Int32` values.
+  private def split_groups_arg(argument text : String) : Array(Int32)
```

Comment:
Why having space separated `argument text`? That's confusing. `numbers_list` is better.

Reply:
agreed for sure